docs/source/releases/latest/read-all-vars-by-default.yaml

```diff
@@ -0,0 +1,133 @@
+refactor:
+- title: 'Apply isort'
+  description: 'isort is a tool for sorting imports to match python standards; this change applies it to this repo.'
+  files:
+    deleted:
+    - ''
+    moved:
+    - ''
+    added:
+    - ''
+    modified:
+    - 'geoips_clavrx/plugins/modules/colormappers/cmap_cldFraction.py'
+    - 'geoips_clavrx/plugins/modules/colormappers/cmap_cldMask.py'
+    - 'geoips_clavrx/plugins/modules/colormappers/cmap_cldMask.py'
+    - 'geoips_clavrx/plugins/modules/colormappers/cmap_cldOpd.py'
+    - 'geoips_clavrx/plugins/modules/colormappers/cmap_cldPhase.py'
+    - 'geoips_clavrx/plugins/modules/colormappers/cmap_cldType.py'
+    - 'geoips_clavrx/plugins/modules/readers/clavrx_hdf4.py'
+    - 'pyproject.toml'
+  related-issue:
+    number: null
+    repo_url: ''
+  date:
+    start: "Oct 30, 2024"
```

Comment:
isort is the only change that is called out in this release note.  Seems like there are A LOT of changes in this PR, including dev container, refactoring code, black updates ?  I would like a little more clarity on what's changed.

Reply:
Oh woops! Thanks mindy. I forgot to add the `enhancement` portion. I'll add that and then ping you again.

Reply:
Fixed.